src/components/DataTypes/date-time-helpers.tsx

```diff
@@ -0,0 +1,24 @@
+import memoize = require('lodash/memoize');
+
+const pad = (n: number) => (n < 10 ? `0${n}` : n);
+
+export const getDefaultDate = (): string => {
+	const now = new Date();
+	const day = pad(now.getDate());
+	const month = pad(now.getMonth() + 1);
+	return `${now.getFullYear()}-${month}-${day}T10:00`;
+};
+
+// Normalize a timestamp to a RFC3339 timestamp, which is required for JSON
+// schema.
+export const normalizeDateTime = memoize((timestamp: string) => {
```

Comment:
This could be changed to `new Date(timestamp).toISOString()`

Reply:
Just a note, theoretically RFC3339 and ISO are slightly different, but I don't think it makes a difference in this case.

Reply:
Removing the milliseconds could be enough in our case: https://stackoverflow.com/questions/522251/whats-the-difference-between-iso-8601-and-rfc-3339-date-formats

_Should be fine™️_

Reply:
Trimming the end of the timestamp should be enough in this case